code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -176,21 +219,32 @@
 		return
 
 	var/staggerslow_stacks = 2
+	var/paralyzesecs = 1
 	var/flavour
-	if(owner.m_intent == MOVE_INTENT_RUN && ( owner.last_move_intent > (world.time - HUNTER_SNEAK_ATTACK_RUN_DELAY) ) ) //Allows us to slash while running... but only if we've been stationary for awhile
+
+	if(movement_affects_sneakattack && owner.m_intent == MOVE_INTENT_RUN && ( owner.last_move_intent > (world.time - HUNTER_SNEAK_ATTACK_RUN_DELAY) ) ) //Allows us to slash while running... but only if we've been stationary for awhile
 		flavour = "vicious"
 	else
-		armor_mod += HUNTER_SNEAK_SLASH_ARMOR_PEN
-		staggerslow_stacks *= 2
+		armor_mod += sneak_attack_armor_pen
 		flavour = "deadly"
 
 	owner.visible_message(span_danger("\The [owner] strikes [target] with [flavour] precision!"), \
 	span_danger("We strike [target] with [flavour] precision!"))
-	target.adjust_stagger(staggerslow_stacks SECONDS)
+	var/datum/action/ability/activable/xeno/hunter_mark/assassin/mark = owner.actions_by_path[/datum/action/ability/activable/xeno/hunter_mark/assassin]
+	if(mark?.marked_target == target)
+		to_chat(owner, span_xenodanger("We strike our death mark with a [flavour], calculated strike."))
+		staggerslow_stacks *= 2
+		paralyzesecs *= 2
+		armor_mod += sneak_attack_armor_pen //should double it.
+	target.adjust_stagger(staggerslow_stacks)
 	target.add_slowdown(staggerslow_stacks)
-	target.ParalyzeNoChain(1 SECONDS)
+	target.ParalyzeNoChain(paralyzesecs SECONDS)
 
-	cancel_stealth()
+	if(disable_on_mob_slash)
+		cancel_stealth()
+	else
```

Comment:
early return

Reply:
again that makes sneak attack have no cooldown and not turn off after attack though.